SquareRoot.py

```diff
@@ -0,0 +1,6 @@
+import math
+import this
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/SquareRoot.py
+++ b/SquareRoot.py
@@ -1,5 +1,6 @@
 import math
 import this
+
 
 def square_root(number):
     sr = math.sqrt(number)
```